crates/hir_expand/src/db.rs

```diff
@@ -395,13 +426,13 @@ fn to_fragment_kind(db: &dyn AstDatabase, id: MacroCallId) -> FragmentKind {
             // If it is root, which means the parent HirFile
             // MacroKindFile must be non-items
             // return expr now.
-            return FragmentKind::Expr;
+            return FragmentKind::Statements;
         }
     };
 
     match parent.kind() {
         MACRO_ITEMS | SOURCE_FILE => FragmentKind::Items,
-        MACRO_STMTS => FragmentKind::Statement,
+        MACRO_STMTS => FragmentKind::Statements,
         ITEM_LIST => FragmentKind::Items,
         LET_STMT => {
             // FIXME: Handle Pattern
```

Comment:
```suggestion
            // FIXME: Handle FragmentKind::Pattern
```
or is something else being implied?